oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentStore.java

```diff
@@ -1350,6 +1358,10 @@ private <T extends Document> BulkUpdateResult sendBulkUpdate(Collection<T> colle
         int i = 0;
         for (UpdateOp updateOp : updateOps) {
             String id = updateOp.getId();
+            int size = updateOp.toString().length();
```

Comment:
Is updateOp the right way to measure this though? Isn't that a generic documentMk side object and what mongo sees is the Bson query below?

Reply:
updateOp's id/value is passed to the Bson query. So, I thought to give a warn here only.

Reply:
I believe the size of UpdateOp doesn't reflect the size of the document. UpdateOp only has the delta, not the actual document size.